mcs/packages/Makefile

```diff
@@ -12,15 +12,63 @@ ROSLYN_FILES = \
 	$(ROSLYN_CSC_DIR)/System.Collections.Immutable.dll	\
 	$(ROSLYN_CSC_DIR)/System.Reflection.Metadata.dll
 
-DISTFILES = $(ROSLYN_FILES)
+ROSLYN_FILES_FOR_MSBUILD = \
+	$(ROSLYN_CSC_DIR)/Microsoft.Build.Tasks.CodeAnalysis.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CSharp.Core.targets \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.CSharp.Scripting.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.CSharp.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.Scripting.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.VisualBasic.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.VisualBasic.Core.targets \
+	$(ROSLYN_CSC_DIR)/Microsoft.Win32.Primitives.dll \
+	$(ROSLYN_CSC_DIR)/System.AppContext.dll \
+	$(ROSLYN_CSC_DIR)/System.Collections.Immutable.dll \
+	$(ROSLYN_CSC_DIR)/System.Console.dll \
+	$(ROSLYN_CSC_DIR)/System.Diagnostics.FileVersionInfo.dll \
+	$(ROSLYN_CSC_DIR)/System.Diagnostics.Process.dll \
+	$(ROSLYN_CSC_DIR)/System.Diagnostics.StackTrace.dll \
+	$(ROSLYN_CSC_DIR)/System.IO.Compression.dll \
+	$(ROSLYN_CSC_DIR)/System.IO.FileSystem.DriveInfo.dll \
+	$(ROSLYN_CSC_DIR)/System.IO.FileSystem.Primitives.dll \
+	$(ROSLYN_CSC_DIR)/System.IO.FileSystem.dll \
+	$(ROSLYN_CSC_DIR)/System.IO.Pipes.dll \
+	$(ROSLYN_CSC_DIR)/System.Reflection.Metadata.dll \
+	$(ROSLYN_CSC_DIR)/System.Security.AccessControl.dll \
+	$(ROSLYN_CSC_DIR)/System.Security.Claims.dll \
+	$(ROSLYN_CSC_DIR)/System.Security.Cryptography.Algorithms.dll \
+	$(ROSLYN_CSC_DIR)/System.Security.Cryptography.Encoding.dll \
+	$(ROSLYN_CSC_DIR)/System.Security.Cryptography.Primitives.dll \
+	$(ROSLYN_CSC_DIR)/System.Security.Cryptography.X509Certificates.dll \
+	$(ROSLYN_CSC_DIR)/System.Security.Principal.Windows.dll \
+	$(ROSLYN_CSC_DIR)/System.Text.Encoding.CodePages.dll \
+	$(ROSLYN_CSC_DIR)/System.Threading.Thread.dll \
+	$(ROSLYN_CSC_DIR)/System.ValueTuple.dll \
+	$(ROSLYN_CSC_DIR)/System.Xml.ReaderWriter.dll \
+	$(ROSLYN_CSC_DIR)/System.Xml.XPath.XDocument.dll \
+	$(ROSLYN_CSC_DIR)/System.Xml.XPath.dll \
+	$(ROSLYN_CSC_DIR)/System.Xml.XmlDocument.dll \
+	$(ROSLYN_CSC_DIR)/VBCSCompiler.exe \
+	$(ROSLYN_CSC_DIR)/VBCSCompiler.exe.config \
+	$(ROSLYN_CSC_DIR)/csi.exe \
+	$(ROSLYN_CSC_DIR)/csi.exe.config \
+	$(ROSLYN_CSC_DIR)/csi.rsp \
+	$(ROSLYN_CSC_DIR)/vbc.exe \
+	$(ROSLYN_CSC_DIR)/vbc.exe.config \
+	$(ROSLYN_CSC_DIR)/vbc.rsp
```

Comment:
I don't know why vbc is needed here and we don't bundle it because it's not usable on Mono. IIRC we are missing VB infrastructure assemblies (or are mostly not working).